mlflow/utils/async_logging/__init__.py

```diff
@@ -1 +1,8 @@
+from mlflow.environment_variables import MLFLOW_ENABLE_ASYNC_LOGGING
 from mlflow.utils.async_logging import run_operations  # noqa: F401
+
+
+def enable_async_logging(enable=True):
+    """Enable or disable async logging globally."""
```

Comment:
`enable_async_logging` with an `enable` arg is a bit redundant / confusing, but I see the point that we need a way for users to *disable* async logging.

What about `mlflow.set_logging_config(synchronous)`?

We can later expand this to include configs for batching, flush frequency, sampling, etc.

By the way, when async logging is disabled, do we shut down the async logging queue / processor?

Reply:
nice call, shall we do something like `mlflow.config.log_asynchronously()`?  I am feeling having a middle layer `config` might be even cleaner.